pegasus/helpers/section_api_helpers.rb

```diff
@@ -113,9 +113,9 @@ def self.completed_levels(user_id)
       and("best_result >= #{ActivityConstants::MINIMUM_PASS_RESULT}")
   end
 
-  VALID_GENDERS = %w(m f)
+  @@valid_genders = %w(m f)
```

Comment:
hm, I assume you already tried getting rid of these warnings by not explicitly including this file in the test? If they are already defined it implies the file was already loaded as a dependency?